examples/3node-withtraffic.pb.txt

```diff
@@ -51,35 +51,6 @@ nodes: {
     name: "ate1"
     type: IXIA_TG
     version: "0.0.1-9999"
```

Comment:
be consistent on how you are representing the nodes for version and exposed ports

Reply:
https://github.com/google/kne/pull/138/commits/f322e580c4df8273520c0f98be4dc6397b568ef0